doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
```

Comment:
VxWorks is a trademark of WindRiver, so add an asterisk  ``VxWorks*`` on this first reference and adjust the title underline.  We don't need to add an asterisk anywhere else in this document.

Reply:
Will do.